src/test/operate.test.js

```diff
@@ -0,0 +1,37 @@
+import operate from '../logic/operate';
+
+test('operate function exists', () => {
+  expect(operate).toBeDefined();
+});
+
+test('return type of operate is a string', () => {
+  expect(typeof operate('12', '13', '+')).toBe('string');
+});
+
+test('operate function adds numbers', () => {
+  expect(operate('12', '13', '+')).toBe('25');
+});
```

Comment:
Let's use "it adds two positive numbers" instead. Do this for other test examples.

Reply:
I'll improve on this